eclair-core/src/test/scala/fr/acinq/eclair/io/PeerSpec.scala

```diff
@@ -91,9 +91,16 @@ class PeerSpec extends FixtureSpec {
       }
     }
 
-    val peer: TestFSMRef[Peer.State, Peer.Data, Peer] = TestFSMRef(new Peer(aliceParams, remoteNodeId, wallet, FakeChannelFactory(channel), switchboard.ref))
+    object MockLimiter extends Actor {
+      def receive: Receive = {
+        case msg: PendingChannelsRateLimiter.AddOrRejectChannel => msg.replyTo ! PendingChannelsRateLimiter.AcceptOpenChannel
+      }
+    }
+    val mockLimiter = system.actorOf(Props(MockLimiter)).ref
```

Comment:
You should use a `TestProbe` on which you `setAutopilot` for this kind of use-case. You should be able to find examples of this across the codebase.

Reply:
Fixed in [5a7b110](https://github.com/ACINQ/eclair/pull/2552/commits/5a7b11076a9de8f644b143755384df1abe4b000c).